launchable/commands/record/commit.py

```diff
@@ -54,8 +57,25 @@ def commit(ctx, source: str, executable: bool, max_days: int, scrub_pii: bool, i
         _import_git_log(import_git_log_output, ctx.obj)
         return
 
+    tracking_client = TrackingClient(Tracking.Command.COMMIT, app=ctx.obj)
+    client = LaunchableClient(tracking_client=tracking_client, app=ctx.obj)
+
+    # Commit messages are not collected in the default.
+    is_collect_message = False
+    try:
+        res = client.request("get", "commits/collect/options")
+        res.raise_for_status()
+        is_collect_message = res.json().get("commitMessage", False)
+    except Exception as e:
+        tracking_client.send_error_event(
+            event_name=Tracking.ErrorEvent.INTERNAL_CLI_ERROR,
+            stack_trace=str(e),
+            api="commits/options",
+        )
+        client.print_exception_and_recover(e)
```

Comment:
Note For Me:
Even if errors happened when request to `commits/collect/options` API, only is_collect_message flag value will be `False`